andela/strikers-ah-frontend
# Ft view reported articles 166733857

#### What does this PR do?
This PR helps an Admin to view reported articles.
#### Description of Task to be completed?
- Created a component to view reported articles
-Added tests for this feature.
#### How should this be manually tested?
- Clone the repo from https://github.com/andela/strikers-ah-frontend.git
- Got to where you cloned the repo and open a terminal/cmd there.
- Checkout to `develop` branch by typing `git checkout develop`.
- Then start the server with `npm start`.
- The home page will open up in your default browser and then you can check the reported article under the popup of profile.
<img width="1239" alt="Screen Shot 2019-06-24 at 00 59 09" src="https://user-images.githubusercontent.com/37753967/59983083-8ddf0f00-961b-11e9-8d4f-8c326ad3ab2b.png">

#### What are the relevant pivotal tracker stories?
#166733857
#### Screenshots (if appropriate).
<img width="1440" alt="Screen Shot 2019-06-24 at 00 49 03" src="https://user-images.githubusercontent.com/37753967/59983015-c7634a80-961a-11e9-9ec8-73d1f243f453.png">
